metrics/metrics_test.go

```diff
@@ -1,13 +1,20 @@
-package metrics_test
+package metrics
```

Comment:
Please revert this to `metrics_test`. So that we focus the tests on exported functions and the behaviour of the package.

```suggestion
package metrics_test
```

Reply:
done!